src/js/components/Form/stories/ValidateOnBlur.js

```diff
@@ -39,7 +40,7 @@ export const ValidateOnBlur = () => (
         />
 
         <FormField label="Email" name="email" required>
-          <TextInput name="email" type="email" />
+          <TextInput name="email" aria-label="email" type="email" />
```

Comment:
How come sometimes we put `aria-label` on the FormField and other times we put it on the TextInput?

Reply:
From WCAG docs, Th best practice is to use it on the actual input control https://www.w3.org/WAI/tutorials/forms/labels/

Reply:
Can we automatically set `aria-label` to `name` if not provided?

Reply:
Agreed on not wanting to clutter the DOM. Can we check for `aria-describedby` though? In other words, if it appears that the caller is managing aria attributes then we don't add any by default, otherwise we do.

Reply:
We can. I suspect it might not be sustainable though, what happens if it is not just `aria-describedby` and its `aria-description` or `aria-labeledBy`? would we still use `aria-label`? There might be other props to consider as well. 
I guess my question is, is it worth adding all of this logic and make the assumption that `aria-label` is the right prop? I need time to reflect on the answer.

Reply:
Regardless, I don't think it should block the change in this story.

Reply:
OK. We can work on that aspect later.

Reply:
Issue created to explore this https://github.com/grommet/grommet/issues/6483